onnxruntime/python/tools/transformers/models/whisper/requirements.txt

```diff
@@ -0,0 +1,19 @@
+# NOTE: In addition to the below packages, you will need to install `ffmpeg` on your machine.
+# Visit https://ffmpeg.org/ for details. You can also install it natively using package managers.
+# Ex:
+# Linux: sudo apt-get install ffmpeg
+# MacOS: sudo brew install ffmpeg
+# Windows: Download from website
```

Comment:
Suggest to add an `installation` section in README about the installation, and add a link to README in this requirement.txt.